### PR TITLE
Extend GetCreateMeta method to retrieve meta data for all projects

### PR DESCRIPTION
**Pull request by nguyen970**
Changed GetCreateMeta method to not require a projectKey. When a user passes in an empty string, it will retrieve the metadata for all projects.

Also changed "Subtasks" to "Subtask" to match Jira's data structure.

### CODE REVIEW

File: metaissue.go [hunk 1]
```diff
@@ -36,15 +36,22 @@ type MetaIssueType struct {
 	Description string                `json:"description,omitempty"`
 	IconUrl     string                `json:"iconurl,omitempty"`
 	Name        string                `json:"name,omitempty"`
-	Subtasks    bool                  `json:"subtask,omitempty"`
+	Subtask     bool                  `json:"subtask,omitempty"`
```
Comment (rbriski) [inline metaissue.go]: Why did you change this?

File: metaissue.go [hunk 2]
```diff
@@ -36,15 +36,22 @@ type MetaIssueType struct {
...
 	Expand      string                `json:"expand,omitempty"`
 	Fields      tcontainer.MarshalMap `json:"fields,omitempty"`
 }
 
 // GetCreateMeta makes the api call to get the meta information required to create a ticket
 func (s *IssueService) GetCreateMeta(projectkey string) (*CreateMetaInfo, *Response, error) {
 
-	apiEndpoint := fmt.Sprintf("rest/api/2/issue/createmeta?projectKeys=%s&expand=projects.issuetypes.fields", projectkey)
+	apiEndpoint := ""
```
Comment (rbriski) [inline metaissue.go]: Please use `net/url` for modifying URLs, including query parameters.

File: metaissue.go [hunk 3]
```diff
@@ -36,15 +36,22 @@ type MetaIssueType struct {
...
+	if len(projectkey) > 0 {
+		apiEndpoint = fmt.Sprintf("rest/api/2/issue/createmeta?projectKeys=%s&expand=projects.issuetypes.fields", projectkey)
+	}else {
+		apiEndpoint = fmt.Sprintf("rest/api/2/issue/createmeta?expand=projects.issuetypes.fields")
+	}
+
+	fmt.Println(apiEndpoint)
```
Comment (rbriski) [inline metaissue.go]: Please remove this.

File: metaissue.go (hunk 1 above)
Comment (nguyen970) [reply metaissue.go]: I changed Subtasks to Subtask to match the jira data model. In addition, it is a boolean to check if the Issue Type is a subtask...so the 's' makes it confusing as it is not subtasks itself.

Comment (rbriski) [reply metaissue.go]: But that will break this library for everyone else that's ever used it.

**Comment by nguyen970** [comment]
I took the same approach as @shdunning with extending the GetList() function in projects. His [commit](https://github.com/andygrunwald/go-jira/commit/2267024e0373d8f74858de437e5b92f8a597be14).
